bindings/node.js/runnable.ts

```diff
@@ -0,0 +1,42 @@
+"use strict";
+
+console.log("testing...");
+
+import { Blst } from "./bindings";
+const blst: Blst = require("blst");
+
+const msg = "assertion"; // this what we're signing
+const DST = "MY-DST"; // domain separation tag
+
+const SK = new blst.SecretKey();
+SK.keygen("*".repeat(32));
+
+////////////////////////////////////////////////////////////////////////
+// generate public key and signature
+
+const pk = new blst.P1(SK);
+const pk_for_wire = pk.serialize();
+
+const sig = new blst.P2();
```

Comment:
As a general remark. While declaring `sig`, being effectively a pointer, `const` would supposedly work, `hash_to` and `sign_with` do modify the internal value of the object. In sense that calls to serialize would yield different values after each step.

Reply:
The `const` here just refers to the pointer to the P2 instance. It does not prevent the instance from mutating itself.